### DISCUSSION TITLE
Allow saving working directory for response file dialog

### DISCUSSION THREAD

**Discussion by ellizio** [mockoon/mockoon#1607] [Ideas]
### Problem

It's annoying to select response file through all directories for each route

### Solution

1. Set `saveWorkingDir` as `true`
https://github.com/mockoon/mockoon/blob/main/packages/desktop/src/renderer/app/components/environment-routes/environment-routes.component.ts#L242

2. Add the corresponding flag to the settings as checkbox

**Comment by 255kb** [comment]
The way the file browsing works is to start from the same directory where you opened the last environment, as files are resolved relatively to the environment's file location. 
I am not sure waht you suggest. Do you suggest saving the file browsing last position as a separated setting?
